bus/api.go

```diff
@@ -190,3 +190,11 @@ type MigrationContract struct {
 	HostIP  string               `json:"hostIP"`
 	ID      types.FileContractID `json:"id"`
 }
+
+// UploadParams contains the metadata needed by a worker to upload an object.
+type UploadParams struct {
+	CurrentHeight uint64
```

Comment:
Isn't it a bit awkward to have consensus logic in the upload endpoint? What happens if the height changes during the upload?